src/status_im/contexts/communities/discover/style.cljs

```diff
@@ -7,7 +7,8 @@
   {:height            56
    :padding-vertical  12
    :justify-content   :center
-   :margin-horizontal 20})
+   :margin-horizontal 20
+   :margin-top        (when platform/ios? 44)})
```

Comment:
should be safe area top, no?

Reply:
It's not the safe are region. It's space between the header and the text here:

<img width="242" alt="Screenshot 2024-01-26 at 11 50 45" src="https://github.com/status-im/status-mobile/assets/45393944/8c3e0b94-14f9-4d25-b3ab-92552eb663f3">

Reply:
But why is that spacing different on android than iOS?

Reply:
The screen itself is positioned absolutely and has some platform specific styling as seen here: https://github.com/status-im/status-mobile/blob/a6cb1c943c847bf7d7df4711a542aba4720af394/src/status_im/contexts/communities/discover/style.cljs#L64-L71

But you are right. I would try refactoring it to remove this altogether.

Reply:
Is it necessary for the header to be absolutely positioned? I guess it's for blurring the background, but can't the background be absolute instead so we avoid conditions these conditions in many places?

Reply:
Right and so that is why I'm pretty sure the spacing caused by safe area is needed there.you're just not directly using it for the "safe-area" but that is what the measurement is based on

Reply:
@clauxx The screen is the one that is positioned absolutely rather than the header

Reply:
Refactored the screen to use safe area and removed platform specific code